.github/CODEOWNERS

```diff
@@ -93,6 +93,7 @@ extension/encoding/textencodingextension/                @open-telemetry/collect
 extension/encoding/zipkinencodingextension/              @open-telemetry/collector-contrib-approvers @MovieStoreGuy @dao-jun
 extension/headerssetterextension/                        @open-telemetry/collector-contrib-approvers @jpkrohling
 extension/healthcheckextension/                          @open-telemetry/collector-contrib-approvers @jpkrohling
+extension/healthcheckextensionv2/                        @open-telemetry/collector-contrib-approvers @mwear
```

Comment:
I'd much rather this replaces the existing extension, than to add a new extension :) 

But i appreciate your comment about making it more difficult to review that way